src/Orchard.Web/Modules/Orchard.Indexing/Services/IndexingTaskExecutor.cs

```diff
@@ -295,7 +295,7 @@ public class IndexingTaskExecutor : IIndexingTaskExecutor, IIndexStatisticsProvi
             try {
                 if (deleteFromIndex.Count > 0) {
                     _indexProvider.Delete(indexName, deleteFromIndex);
-                    Logger.Information("Added content items to index: {0}", addToIndex.Count);
+					Logger.Information("Deleted content items from index: {0}",  deleteFromIndex.Count);
```

Comment:
It seems that you inserted a tab or two. Please use spaces for indentation instead. Thanks